sympy/testing/pytest.py

```diff
@@ -54,6 +63,7 @@ def raises(expectedException, code=None):
         >>> from sympy.testing.pytest import raises
 
         >>> raises(ZeroDivisionError, lambda: 1/0)
+        <ExceptionInfo ZeroDivisionError('division by zero')>
```

Comment:
Maybe this will fix Python 3.5 CI:
```suggestion
        <ExceptionInfo ZeroDivisionError(...)>
```